virtualizor.py

```diff
@@ -105,6 +105,14 @@ def __init__(self, conf):
         self.conn = libvirt.open('qemu+ssh://root@%s/system' %
                                  conf.target_host)
 
+    def find_emulator(self):
```

Comment:
I suggest something like that instead, to avoid the serie of if / else:

````
def find_emulator(self):
    for location in ('/usr/bin/qemu-system-x86_64', '/usr/libexec/qemu-kvm'):
        if self.call('test', '-f', location) == 0:
            return location
````